backend/middlewares/error-middleware.js

```diff
@@ -1,6 +1,6 @@
 import { HTTP_STATUS, RESPONSE_MESSAGES } from '../utils/constants.js';
 
-const errorMiddleware = (err, req, res, next) => {
+const errorMiddleware = (err, req, res) => {
   console.error(err.stack);
```

Comment:
We can't do this, because the default error middleware which the express figures out is supposed to have 4 params, We need to compress just on this line or you can call next() which anyways doesn't make much sense

Reply:
done sir